pvlib/atmosphere.py

```diff
@@ -450,6 +457,11 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
     _coefficients['polysi'] = (
         0.84090, -0.027539, -0.0079224, 0.13570, 0.038024, -0.0021218)
     _coefficients['multisi'] = _coefficients['polysi']
+    _coefficients['cigs'] = (
+        0.85252, -0.022314, -0.0047216, 0.13666, 0.013342, -0.0008945)
+    _coefficients['asi'] = (
+        1.12094, -0.047620, -0.0083627, -0.10443, 0.098382,-0.0033818)
+	
```

Comment:
This empty-looking line has a tab character. You can just delete the whole line.